cabal-install/changelog

```diff
@@ -1,6 +1,13 @@
 -*-change-log-*-
 
 3.10.2.0 Hécate <hecate+github@glitchbra.in> August 2023
```

Comment:
This line got duplicated in this forwardport to master and it's there to this very day. :)

Reply:
I saw that, but then saw it has a completely different date so I'm not really sure what's going on.

Reply:
Well well well

Reply:
#10149